### PR TITLE
Blog listing and pagination styling

### PR DESCRIPTION
**Pull request by helenb**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1347772512)

### Description of Changes Made

- Adds styling for the blog listing and pagination
- Changes the image crop used for the avatar
- Renames `title-block.html` to `title-filters.html`
- Removes unnecessary extra template for the blog listing
- Uses the re-usable `listing--avatar.html` template for the blog item
- Note that the pagination is hard-coded and the mobile version (with fewer numbers) isn't in place yet - waiting for back-end.

### How to Test
Can be viewed in wagtail: http://localhost:8000/blog/
Or the pattern library: http://localhost:8000/pattern-library/pattern/patterns/pages/blog/blog_listing.html

### Screenshots

<details>
  <summary>Expand to see more</summary>

Screenshot 2024-01-19 at 15 07 04

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Tests and linting passes.

#### Unit tests

- [ ] Added
- [x] Not required

#### Documentation

- [ ] Updated build docs
- [ ] Updated editor guidelines (https://docs.google.com/document/d/1PAWccdQ4tfaZsrEWmpDhvP3GH5RRmBOARFVp4b-kje8/edit?usp=sharing)
- [x] Not required

#### Browser testing

- [x] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android
- [ ] Not required

#### Data protection

- [x] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Accessibility

- [x] Automated WCAG 2.1 tests pass
- [x] Manual WCAG 2.1 tests completed
- [x] I have tested in a screen reader
- [x] I have tested in high-contrast mode
- [x] Any animations removed for prefers-reduced-motion
- [ ] Not required

#### Sustainability

- [x] Images are optimised and lazy-loading used where appropriate
- [ ] SVGs have been optimised
- [x] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [x] New font variants have not been added
- [ ] Not required

#### Pattern library

- [x] The pattern library component for this template displays correctly, and does not break parent templates
- [x] The styleguide is updated if relevant
- [ ] Changes are not relevant the pattern library

### CODE REVIEW

File: tbx/blog/models.py
Comment (helenb) [inline tbx/blog/models.py:83]: @engineervix are these changes ok to the model?

File: tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html
Comment (helenb) [inline tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html:1]: @engineervix this is hard-coded for now - I am hoping you can fix this in your back-end branch.

File: tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.yaml
Comment (helenb) [inline tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.yaml:1]: This needs updating once the back-end is done

File: tbx/project_styleguide/templates/patterns/molecules/title-filters/title-filters.html
Comment (helenb) [inline tbx/project_styleguide/templates/patterns/molecules/title-filters/title-filters.html:1]: @engineervix this one is re-named

File: tbx/project_styleguide/templates/patterns/organisms/blog-listing/blog-listing.html
Comment (helenb) [inline tbx/project_styleguide/templates/patterns/organisms/blog-listing/blog-listing.html:1]: @chris-lawton @engineervix I think this file may have been useful for the js implementation. We don't need it now, and the equivalent ones for work-listing and latest-posts can probably be removed in due course.

File: tbx/blog/models.py
Comment (engineervix) [reply tbx/blog/models.py:83]: @helenb that looks good to me, this is the same as what was done on the `BlogPage`

Comment (helenb) [reply tbx/blog/models.py:83]: ah thanks - I thought I had rebased that so not sure why this is showing as different. I'll rebase again to make sure.

Comment (helenb) [reply tbx/blog/models.py:83]: It is up to date - I think this is showing just because I changed it as well before I rebased once your branch was merged.

File: tbx/project_styleguide/templates/patterns/molecules/listing/listing--avatar.html
Comment (chris-lawton) [inline tbx/project_styleguide/templates/patterns/molecules/listing/listing--avatar.html:16]: Think `date` is missing a value in the pattern library

File: tbx/project_styleguide/templates/patterns/molecules/title-filters/title-filters.html
Comment (chris-lawton) [inline tbx/project_styleguide/templates/patterns/molecules/title-filters/title-filters.html:12]: Should this wrap `<div class="title-filters__tags">` so there won't be an empty div if there are no tags?

File: tbx/static_src/sass/components/_pagination.scss
Comment (chris-lawton) [inline tbx/static_src/sass/components/_pagination.scss:96]: 👏

**Review by chris-lawton — APPROVED** [review]
Looks great @helenb, thanks. Just left a few comments to review.

File: tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html:2]: I'll hold off till the markup is finalised - I'll make a new ticket to finish this off once Victor's work is merged

File: tbx/project_styleguide/templates/patterns/molecules/title-filters/title-filters.html
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/molecules/title-filters/title-filters.html]: good point. I'm not sure we'd use this component if there weren't filters, but good to get the logic correct. Fixed in c0f05047f3b8c32aa16c44c36352644681786abe.

File: tbx/project_styleguide/templates/patterns/molecules/listing/listing--avatar.html
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/molecules/listing/listing--avatar.html:16]: Resolved in 97f3c854c15273d419cd541255168cf81faa0544

**Pull request merged**
